src/core/font_substitutions.js

```diff
@@ -437,7 +437,8 @@ function getFontSubstitution(
       (italic && ITALIC) ||
       NORMAL;
     substitutionInfo = {
-      css: `${loadedName},sans-serif`,
+      css: loadedName,
+      mustGuessFallback: true,
```

Comment:
How about a slightly shorter name?
```suggestion
      guessFallback: true,
```

Reply:
Also, how about including an explicit `guessFallback: false,` line below such that the objects returned by this function always have a consistent "shape"?

https://github.com/mozilla/pdf.js/pull/16416/files#diff-84365a5fa783e74d4c83766a21305541dedb5d8ccefa51df9f95fa3a38bf0cfcR482-R487